src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,8 +1,8 @@
-*REMOVED*override sealed Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode.ToString() -> string
```

Comment:
Reverted. I should learn not to let the PublicAPI files update through the automatic code fix ^,^